cohere/client.py

```diff
@@ -224,14 +224,44 @@ def detect_language(self, texts: List[str]) -> List[Language]:
             results.append(Language(result["language_code"], result["language_name"]))
         return DetectLanguageResponse(results)
 
-    def feedback(self, id: str, feedback: str, accepted: bool):
+    def feedback(self, id: str, good_response: bool, desired_response: str = "", feedback: str = "") -> Feedback:
+        """Give feedback on a response from the Cohere API to improve the model.
+
+        Can be used programmatically like so:
+
+        Example: a user accepts a model's suggestion in an assisted writing setting
+        ```
+        generations = co.generate(f"Write me a polite email responding to the one below:\n{email}\n\nResponse:")
```

Comment:
here and in other examples
```suggestion
        generations = co.generate(f"Write me a polite email responding to the one below:\n{email}\n\nResponse:").generations
```

Reply:
the .generations isn't necessary anymore. We have convenience methods that let you index into the Generations object directly instead of having to access a property